project/src/main/java/com/google/sps/data/PlacesFetcher.java

```diff
@@ -55,16 +59,13 @@
         .apiKey(System.getenv("API_KEY"))
         .build();
 
-    // TODO(M0): add exception handling and testing
     /**
      * Builds a query and requests it from Google Places API.
      *
      * @return an immutable list of places that supply the query.
-     * @throws IOException
-     * @throws InterruptedException
-     * @throws ApiException
+     * @throws FetcherException
```

Comment:
In public API, please explain in what cases the exception is thrown.

Reply:
added it

Reply:
Can't see the documentation?